src/main/java/org/prebid/server/analytics/reporter/greenbids/model/ExplorationResult.java

```diff
@@ -0,0 +1,18 @@
+package org.prebid.server.analytics.reporter.greenbids.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+
+import java.util.Map;
+
+@Value(staticConstructor = "of")
+public class ExplorationResult {
+
+    String fingerprint;
+
+    @JsonProperty("keptInAuction")
+    Map<String, Boolean> keptInAuction;
+
+    @JsonProperty("isExploration")
+    Boolean isExploration;
+}
```

Comment:
I believe the module can just re-use the model from the analytics, so you don't need to have duplicate models

Reply:
Removed duplicated `ExplorationResult`